src/hotspot/share/cds/filemap.hpp

```diff
@@ -237,6 +237,7 @@ class FileMapHeader: private CDSFileMapHeaderBase {
   bool   _use_full_module_graph;        // Can we use the full archived module graph?
   size_t _ptrmap_size_in_bits;          // Size of pointer relocation bitmap
   narrowOop _heap_obj_roots;            // An objArray that stores all the roots of archived heap objects
+
```

Comment:
I think you can revert the changes to this file since there are only couple of blank lines addition/deletion probably by mistake.